docker/ravendb-nanoserver/Dockerfile.2022

```diff
@@ -0,0 +1,32 @@
+# escape=`
```

Comment:
can we change the filename suffix to actual win version suffix `ltsc2022`

Reply:
renamed the Dockerfile to Dockerfile.ltsc2022